### ISSUE TITLE
Login Page

### ISSUE THREAD

**Issue by atxfjrotc** [atxfjrotc/USwap#3]

**Comment by atxfjrotc** [comment]
@erobx Let's both design a login page? That way we both get experience with doing the whole Angular/Material design thing?

**Comment by erobx** [comment]
Yeah we can do that just create two separate branches and we're good

**Comment by atxfjrotc** [comment]
Proposed User Story for this task:

As a **user**, **developer**, and **human being**, I want to be able to **login to the website to use it**.

**Comment by atxfjrotc** [comment]
Suydam Task List:
 - Learn Angular
 - Implement login page

**Comment by atxfjrotc** [comment]
Alrighty, I have a first draft ready on my end.

**Comment by atxfjrotc** [comment]
Given that the first draft is done, and @erobx has ceased his concurrent development of a draft in favor of mine, I am closing this issue and pulling my changes to main.

**Comment by atxfjrotc** [comment]
@erobx why was this reopened?

**Comment by erobx** [comment]
Good question. We should include the signup button within the login page, but that should be a separate issue, since we currently don't have a signup page.
